framework/Metrics/metrics/MetricUtilities.py

```diff
@@ -227,3 +229,15 @@ def _getPDFCommonArea(data1, data2):
   low, high = _getBounds(stats1, stats2)
   return scipy.integrate.quad(lambda x:min(pdf1(x),pdf2(x)),low,high,limit=1000)[0]
 
+def _getSTDReduction(data1, data2):
+  """
+  Gets the uncertainty (in form of standard deviation) reduction from data1 to data2.
+  The closer to 1.0 of the reduction fraction, the better representative the experiment is.
+  @ In, data1, data from the prior distribution
+  @ In, data2, data from the posterior distribution
```

Comment:
please provide the type for 'data1' and 'data2'

Reply:
'a set of data' for both

Reply:
Currently this function is not used